internal/cloud/cloudplan/saved_plan_test.go

```diff
@@ -28,15 +31,69 @@ func TestCloud_loadBasic(t *testing.T) {
 	}
 }
 
-func TestCloud_saveBasic(t *testing.T) {
-	tmp := t.TempDir()
-	bookmarkPath := filepath.Join(tmp, "saved-bookmark.json")
+func TestCloud_loadCheckRunID(t *testing.T) {
+	// Run ID must never be empty
+	file := "./testdata/plan-bookmark/empty_run_id.json"
+	_, err := LoadSavedPlanBookmark(file)
+	if !errors.Is(err, ErrInvalidRunID) {
+		t.Fatalf("expected %s but got %s", ErrInvalidRunID, err)
+	}
+}
+
+func TestCloud_loadCheckHostname(t *testing.T) {
+	// Hostname must never be empty
+	file := "./testdata/plan-bookmark/empty_hostname.json"
+	_, err := LoadSavedPlanBookmark(file)
+	if !errors.Is(err, ErrInvalidHostname) {
+		t.Fatalf("expected %s but got %s", ErrInvalidHostname, err)
+	}
+}
+
+func TestCloud_loadCheckVersionNumberBasic(t *testing.T) {
```

Comment:
This is totally a non-blocker and more FYI. You don't necessarily need to append `Basic`. This is a convention we established in terraform-provider-tfe to distinguish resource tests for required attributes and `Full` for required AND optional attributes. 

This is normally the convention I follow when naming a test for a particular struct, which I think helps organize it a bit:
`Test<SomeStruct>_<the behavior you're testing>`, so in the above case:
`TestSavedPlanBookmark_loadCheckVersionNumber`

`TestCloud_` in this case would indicate this test is regarding the [Cloud](https://github.com/hashicorp/terraform/blob/6fa5784129f706a4b459b4495394899c6cc3e041/internal/cloud/backend.go#L52-L108) struct. Besides organization, it's helpful when passing in the `-run` flag when testing locally (which greps all the test names based on the value provided). So if I wanted to run all the tests relevant to `SavePlanBookmark` I would do:
```sh
go test ./internal/cloud -v -run TestSavePlanBookmark_
```

Again, it's mostly for convenience/organization rather than a hard rule.

Reply:
I will paste this advice in @nfagerlund's channel. Merged ⭐